Samples~/AvatarCreatorSamples/AvatarCreatorWizard/Scripts/UI/Buttons/AvatarButton.cs

```diff
@@ -16,13 +16,15 @@ public class AvatarButton : MonoBehaviour, IPointerEnterHandler, IPointerExitHan
         [SerializeField] private Button customizeButton;
         [SerializeField] private Button selectButton;
 
+        private RectTransform rawImageRectTransform;
         private string avatarId;
         private bool showButtons;
 
         private CancellationTokenSource ctxSource;
 
         private async void Start()
         {
+            rawImageRectTransform = image.GetComponent<RectTransform>();
```

Comment:
What about converting variable to property and defining this inside of the property.